django/db/backends/postgresql/compiler.py

```diff
@@ -0,0 +1,61 @@
+from django.db.models.sql import compiler
+
+SQLCompiler = compiler.SQLCompiler
+SQLDeleteCompiler = compiler.SQLDeleteCompiler
+SQLUpdateCompiler = compiler.SQLUpdateCompiler
+SQLAggregateCompiler = compiler.SQLAggregateCompiler
+
+
+class SQLInsertCompiler(compiler.SQLInsertCompiler):
```

Comment:
I don't like the duplication of code done here. Could we instead branch on database feature in the main compiler class?